.github/workflows/core-dependency-update.yml

```
@@ -59,4 +59,9 @@ jobs:
           echo "📦 Extracted core version: ${CORE_VERSION}"
           
           # Run the complete core dependency update pipeline
-          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}" 
\ No newline at end of file
+          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}" 
+  codecov:
```

actionlint (1.7.7)

"steps" section is missing in job "codecov"

(syntax-check)